### PR TITLE
Master

### PR DESCRIPTION
**Pull request by raphweb**
Some convenience methods in the java interface:
- query the domain of an IntVar
- remove constraint in CpModel
- remove IntVar/IntervalVar in CpModel
The domain query would be really nice :-)

### CODE REVIEW

File: ortools/java/com/google/ortools/sat/CpModel.java [hunk 1]
```diff
@@ -957,6 +957,21 @@ public void addHint(IntVar var, long value) {
     modelBuilder.getSolutionHintBuilder().addVars(var.getIndex());
     modelBuilder.getSolutionHintBuilder().addValues(value);
   }
+
```
Comment (lperron) [inline ortools/java/com/google/ortools/sat/CpModel.java]: can't work.

Protobuf stores indices of variables.

**Comment by lperron** [comment]
Thanks,. I will have a look. The remove part cannot work. I will cherrypick the rest.

**Comment by raphweb** [comment]
@googlebot I fixed it.

**Comment by raphweb** [comment]
Added new pull request with fixed author.

**Pull request closed without merging**